player/lua/ytdl_hook.lua

```diff
@@ -27,6 +27,7 @@ end)
 
 local chapter_list = {}
 local playlist_cookies = {}
+local playlist_title = ""
```

Comment:
`= ""` needs to be removed to not always add this to the json as empty strings are truthy in Lua.

Reply:
Yes, I have actually done this locally because of exactly the issue you're describing.